code/drasil-example/Drasil/GamePhysics/DataDefs.hs

```diff
@@ -367,11 +367,11 @@ momentOfInertia :: QDefinition
 momentOfInertia = mkQuantDef QP.momentOfInertia momentOfInertiaEqn
 
 momentOfInertiaEqn :: Expr
-momentOfInertiaEqn = (sumAll (Atomic "i")((sy massI)*(sy  rRot $^ 2)))
+momentOfInertiaEqn = sumAll (Atomic "i") (sy massI) * (sy rRot $^ 2)
```

Comment:
This change is incorrect, it changes the meaning. This is clearer in the latex/html where there's a variable j outside of its binder.

Reply:
Oh yeah, I think I pushed this change in error, I was just playing around with the Hlint suggestions and I think I hit 'Enter' on scrolling to git push